final_task/rss_reader/json_date_util.py

```diff
@@ -0,0 +1,26 @@
+import json
+
+
+def json_convert(dict_with_info: dict, limit: str):
+    json_date = None
+
+    output = {'Feed': [],
+              'Entries': []
+              }
+    temp = {'Title': [],
+            'Date': [],
+            'Links': [],
+            'Info': []
+            }
+    if type(dict_with_info) == dict:
+        output['Feed'] = dict_with_info['Feed']
+        for index in range(int(len(limit))):
```

Comment:
During our sessions throughout the course we *emphasized multiple times* that you should __not__ use such constructions. Please fix it.